.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required 
+dist: trusty
+language: ruby
+rvm:
+  - "2.4.3"
+  - "2.5.0"
+script: bundle exec rake
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y libunwind8 wget libicu52 libssl-dev curl unzip gettext libcurl4-openssl-dev zlib1g uuid-dev bzip2
+  - wget https://daily-builds.s3.amazonaws.com/RavenDB-4.0.2-linux-x64.tar.bz2; tar -xjf RavenDB-4.0.2-linux-x64.tar.bz2
```

Comment:
we could use latest stable here (need to follow redirect)
https://hibernatingrhinos.com/downloads/RavenDB%20for%20Windows%20x64/latest